deploy/helm/values.schema.json

```diff
@@ -1254,6 +1254,36 @@
         },
         "global": {
             "type": "object"
+        },
+        "operator": {
+            "description": "If enabled, deploys otel-operator to the cluster.",
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "type": "boolean"
+                },
+                "additionalProperties": true
+            }
+        },
+        "opentelemetry-operator": {
```

Comment:
Why do we need the "legacy" fields in the schema?

Reply:
This is supposed to be a workaround for older versions of the Helm tool. We should check if it actually works.

Reply:
I verified that in CI which uses Helm 3.8.0, and it deployed all certmanager and operator pods correctly (https://github.com/solarwinds/swi-k8s-opentelemetry-collector/actions/runs/11818302623/job/32925550157#step:12:8), so `values.yaml` is respected correctly in older Helm: